controllers/permits/special-uses/validate.js

```diff
@@ -36,9 +36,136 @@ const phoneNumber = schema.phoneNumber;
 
 //*******************************************************************
 
+function zipFormat(input){
+	
+	let valid = true;
+
+	if (typeof input === 'number'){
+
+		const inputStr = input + '';
+
+		if (!inputStr.match(/^[0-9]{5}$|^[0-9]{9}$/)){
+
+			valid = false;
+
+		}
+
+	}
+	return valid;
```

Comment:
@cameronwolf this will return true is input is not a number.  If passed a string (eg "ABCD") it would validate as true.